tests/models.py

```diff
@@ -244,3 +268,67 @@ def test_depends_on_past(self):
             ignore_first_depends_on_past=True)
         ti.refresh_from_db()
         self.assertEqual(ti.state, State.SUCCESS)
+
+    @parameterized.expand([
+        #
```

Comment:
Could you add a header comment here listing all the fields, maybe abbreviated? I know they're in the function signature but at first glance it's hard to tell which of the numbers (`5, 0, 0, 0, 0, 0`) goes with which field. A header would help.